app/services/g6importService.py

```diff
@@ -0,0 +1,62 @@
+import json
+import jsonschema
+from jsonschema import validate
```

Comment:
These imports are a bit strange. I'd recommend adding `ValidationError` to the second import and then try catch blocks later in the module.